packages/mantine/package.json

```
@@ -0,0 +1,107 @@
+{
+  "name": "@rjsf/mantine",
+  "version": "6.0.0-beta.1",
+  "main": "dist/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "type": "module",
+  "description": "Mantine theme, fields and widgets for react-jsonschema-form",
+  "exports": {
+    ".": {
+      "require": "./dist/index.js",
+      "import": "./lib/index.js",
+      "types": "./lib/index.d.ts"
```

A warning happens when `types` isn't first
```suggestion
      "types": "./lib/index.d.ts",
      "require": "./dist/index.js",
      "import": "./lib/index.js"
```